source/installation/upgrade-minio.rst

```diff
@@ -10,67 +10,58 @@ Upgrade a MinIO Deployment
+MinIO operations are atomic and strictly consistent, such that applications using MinIO or S3 SDKs can rely on the built-in :aws-docs:`transparent retry <general/latest/gr/api-retries.html>` without further client-side logic.
+This ensures upgrades are non-disruptive to ongoing operations.
 
-       **Never** perform blind upgrades on production deployments.
+"Rolling" or serial "one-at-a-time" upgrade methods do not provide any advantage over the recommended "parallel" procedure, and can introduce unnecessary complexity to the upgrade procedure.
+For virtualized environments which *require* rolling updates, you should amend the recommend procedure as follows:
 
-   * - Upgrade Only When Necessary
-     - MinIO follows a rapid development model where there may be multiple
-       releases in a week. There is no requirement to follow these updates
-       if your deployment is otherwise stable and functional.
+1. Update the MinIO Binary in the virtual machine or container one at a time.
+2. Restart the MinIO deployment using :mc-cmd:`mc admin service restart`.
+3. Update the virtual machine/container configuration to use the matching newer MinIO image.
+4. Perform the rolling restart of each machine/container with the updated image.
 
-       Upgrade only if there is a specific feature, bug fix, or other
-       requirement necessary for your workload. Review the 
-       :minio-git:`Release Notes <minio/releases>` for each Server release
-       between your current MinIO version and the target version.
+Check Release Notes
+~~~~~~~~~~~~~~~~~~~
 
-   * - Upgrades require Simultaneous Restart 
-     - Ensure your preferred method of node management supports operating on 
-       all nodes simultaneously.
-     
-       .. include:: /includes/common-installation.rst
-          :start-after: start-nondisruptive-upgrade-desc
-          :end-before: end-nondisruptive-upgrade-desc
+MinIO publishes :minio-git:`Release Notes <minio/releases>` for your reference as part of identifying the changes applied in each release.
+Review the associated release notes between your current MinIO version and the newer release such that you have a complete view of any changes.
+
+Pay particular attention to any releases that are backwards incompatible.
+You cannot trivially downgrade from any such release.
 
 .. _minio-upgrade-systemctl:
 
 Update ``systemctl``-Managed MinIO Deployments
 ----------------------------------------------
 
-Deployments managed using ``systemctl``, such as those created
-using the MinIO :ref:`DEB/RPM packages <deploy-minio-distributed-baremetal>`,
-require manual update and simultaneous restart of all nodes in the
-MinIO deployment.
+Use these steps to upgrade a MinIO deployment where the MinIO server process is managed by ``systemctl``, such as those created using the MinIO :ref:`DEB/RPM packages <deploy-minio-distributed-baremetal>`.
 
 1. **Update the MinIO Binary on Each Node**
```

Comment:
We don't typically bold procedure steps. I wouldn't change the normal style just for this procedure.
Let's let our CSS handle that.

Reply:
Done